docs/reference/upgrade/rolling_upgrade.asciidoc

```diff
@@ -10,8 +10,8 @@ running the older version.
 Rolling upgrades are supported:
 
 * Between minor versions
-* {stack-ref-67}/upgrading-elastic-stack.html[From 5.6 to 6.7]
-* {stack-ref-70}/upgrading-elastic-stack.html[From 6.7 to 7.0]
+* {stack-ref-67}/upgrading-elastic-stack.html[From 5.6 to 6.8]
```

Comment:
`{stack-ref-67}` points to the 6.7 branch of the [Elastic Stack Installation and Upgrade Guide](https://www.elastic.co/guide/en/elastic-stack/6.7/index.html).

I opened elastic/docs#956 to add a `{stack-ref-68} attribute. However, if you don't want to wait or have the dependency, you can just the `https://www.elastic.co/guide/en/elastic-stack/6.8/upgrading-elastic-stack.html` link.